plugins/uefi/fu-plugin-uefi.c

```diff
@@ -582,6 +595,41 @@ fu_plugin_uefi_test_secure_boot (FuPlugin *plugin)
 	fu_plugin_add_report_metadata (plugin, "SecureBoot", result_str);
 }
 
+static gboolean load_config_file (FuPlugin *plugin, GError **error)
```

Comment:
I'm not super keen on this. I'd be okay with something abstract like `gchar *fu_plugin_get_setting(plugin, const gchar *key)` that automatically loads the right config file -- and then it stops each plugin having a different version of the same thing. The key file system works now, but we might want a different config system in the future, hence why I'd like to abstract it out a little.

Reply:
Adjusted (and for your other comment that got lost in the force push)